pysb/examples/gpu_ssa/run_earm_gpu_ssa.py

```diff
@@ -0,0 +1,46 @@
+import logging
```

Comment:
Example model filenames should clarify which GPU SSA they're using. A short docstring at the top describing the example would be helpful, too.